docs/source/mclda.rst

```diff
@@ -140,22 +140,36 @@ One can use ``fit`` to perform multi-class LDA over a set of data:
 
     **Keyword arguments:**
 
-    =========== =============================================================== ====================
-      name         description                                                   default
-    =========== =============================================================== ====================
-     method     The choice of methods:                                           ``:gevd``
-
-                - ``:gevd``: based on generalized eigenvalue decomposition
-                - ``:whiten``: first derive a whitening transform from ``Sw``
-                  and then solve the problem based on eigenvalue
-                  decomposition of the whiten ``Sb``.
-    ----------- --------------------------------------------------------------- --------------------
-     outdim     The output dimension, *i.e* dimension of the transformed space   ``min(d, nc-1)``
-    ----------- --------------------------------------------------------------- --------------------
-     regcoef    The regularization coefficient.                                  ``1.0e-6``
-                A positive value ``regcoef * eigmax(Sw)`` is added to the
-                diagonal of ``Sw`` to improve numerical stability.
-    =========== =============================================================== ====================
+    ================= =============================================================== ========================
+      name                      description                                                         default
+    ================= =============================================================== ========================
+     method           The choice of methods:                                           ``:gevd``
+
+                        - ``:gevd``: based on generalized eigenvalue decomposition
+                        - ``:whiten``: first derive a whitening transform from ``Sw``
+                          and then solve the problem based on eigenvalue
+                          decomposition of the whiten ``Sb``.
+    ----------------- --------------------------------------------------------------- ------------------------
+     outdim           The output dimension, *i.e* dimension of the transformed space   ``min(d, nc-1)``
+    ----------------- --------------------------------------------------------------- ------------------------
+     regcoef          The regularization coefficient.                                  ``1.0e-6``
+                      A positive value ``regcoef * eigmax(Sw)`` is added to the
+                      diagonal of ``Sw`` to improve numerical stability.
+    ----------------- --------------------------------------------------------------- ------------------------
+     covestimatorSb   Custom covariance estimator for between-class covariance.        ``SampleCovariance()``
```

Comment:
Argument names should be lowercase. Also the `covestimator` argument should be documented.

A possible solution would be to use a single argument which would take a 2-tuple for models where that makes sense.

Also, "S**i**mpleEstimator".

Reply:
OK, thanks. A 2-tuple can be a bit confusing though. Would underscores be acceptable, for example `covestimator_within` and `covestimator_between`?

And which `covestimator` argument isn't documented?

Reply:
Underscores are OK when names are getting too long, but better avoid them when possible. I guess it's fine if we can't find anything better...



> And which `covestimator` argument isn't documented?

Sorry, I thought this was also supposed to cover `LinearDiscriminant`, which takes `covestimator` as an argument.

Reply:
I've changed names to `covestimator_within` and `covestimator_between`.